scss/components/_button.scss

```diff
@@ -113,11 +111,13 @@ $button-responsive-expanded: false !default;
     padding: $button-padding;
   }
 
-  -webkit-appearance: none;
   border: $button-border;
   border-radius: $button-radius;
   transition: $button-transition;
+  font-family: $button-font-family;
   font-size: map-get($button-sizes, default);
+  font-weight: $button-font-weight;
+  -webkit-appearance: none; // sass-lint:disable-line no-vendor-prefixes
```

Comment:
Do we need `-webkit-appearance` instead of `appearance` here ? Autoprefixer supports it now.